test/system/navbar_policy_visibility_test.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require "test_helper"
+
+class NavbarPolicyVisibilityTest < ActionDispatch::IntegrationTest
```

Comment:
Because it's a system test, it failed on the system level on the CI. 

Maybe you missed that because you ran the `rails test`, but didn't run the `rails test:system`

```
NavbarPolicyVisibilityTest#test_admin_sees_Admin_if_permitted:
ActiveRecord::RecordInvalid: Validation failed: Classroom can't be blank
    test/system/navbar_policy_visibility_test.rb:8:in 'NavbarPolicyVisibilityTest#setup'
```

What I'm thinking is that we can create something similar to what you did for PortfolioPolicy and PortfolioPolicyTest, rather than creating a system test for that. 

What do you think?